static/app/components/replays/breadcrumbs/replayComparisonModal.tsx

```diff
@@ -33,14 +39,18 @@ export default function ReplayComparisonModal({
             Hydration Error
             <FeatureBadge type="beta" />
           </h4>
-          <GithubFeedbackButton
-            href="https://github.com/getsentry/sentry/discussions/62097"
-            label={t('Discussion')}
-            title={null}
-            analyticsEventKey="replay.details-hydration-discussion-clicked"
-            analyticsEventName="Replay Details Hydration Discussion Clicked"
-            priority="primary"
-          />
+          {pauseFocusTrap && unpauseFocusTrap ? (
+            <FeedbackWidgetButton
+              optionOverrides={{
+                onFormOpen: () => {
+                  pauseFocusTrap();
+                },
+                onFormClose: () => {
+                  unpauseFocusTrap();
+                },
```

Comment:
This can be compressed. Instead of wrapping with an extra function call, we can just pass the functions themselves:

```suggestion
                onFormOpen: pauseFocusTrap,
                onFormClose: unpauseFocusTrap,
```

Reply:
The functions return focustrap though, while onformclose needs to return void.

Reply:
ah, good point!